src/sentry/models/release.py

```diff
@@ -71,13 +71,15 @@ def get_cache_key(cls, project_id, version):
 
     @classmethod
     def get(cls, project, version):
+        # TODO: should this be cached by organization now?
```

Comment:
yeah so i guess the only question i had here was how to correctly bust the cache since it'd be possible to return the wrong release if the only thing we changed was switching project_id and org_id, right? should i just modify `get_cache_key` so that the base key is different? so something like `'release:3:%s:%s'`?